core/validation/messages.py

```diff
@@ -39,3 +41,18 @@
 )
 # PRE-TRANSFORMATION
 BLANK_SIGN_OFF = "In tab '8 - Review & Sign-Off', cell {cell} is blank but is required."
+
+# PRE-VALIDATION
+PRE_VALIDATION_MESSAGES = {
+    "Fund Type": "Cell E7 in the “project admin” must contain a fund type from the dropdown list provided."
+    " Do not enter your own content.",
+    "Place Name": "Cell E8 in the “project admin” must contain a place name from the dropdown list provided. "
+    "Do not enter your own content.",
+    "Reporting Period": "Cell B6 in the “start here” tab must say “1 April 2023 to 30 September 2023”. Select this "
+    "option from the dropdown list provided.",
+    "Not Excel File": "The selected file must be an Excel file",
+    "Wrong Template": "The selected file must be the Town Deals and Future High Streets Fund Reporting Template"
+    f" ({TF_ROUND_4_TEMPLATE_VERSION})",
+    "Form Version": "You have submitted the wrong reporting template. Make sure you submit Town Deals and Future High "
```

Comment:
The "You have submitted the wrong reporting template" pat of this message isn't in the current version from design unless I'm mistaken. I'm looking at SMD-233 and it seems to be removed

Reply:
good spot looks like it should be "The selected file must be the Town Deals and Future High Streets Fund Reporting Template ({VERSION})"